source/numerics.tex

```diff
@@ -3917,8 +3917,8 @@
 \indextext{engines with predefined parameters!\idxcode{mt19937}}%
 \begin{itemdecl}
 using mt19937 =
-      mersenne_twister_engine<uint_fast32_t,
-       32,624,397,31,0x9908b0df,11,0xffffffff,7,0x9d2c5680,15,0xefc60000,18,1812433253>;
+      mersenne_twister_engine<uint_fast32_t, 32, 624, 397, 31,
+       0x9908'b0df, 11, 0xffff'ffff, 7, 0x9d2c'5680, 15, 0xefc6'0000, 18, 1'812'433r'253>;
```

Comment:
A stray `r` has appeared in the last number here.

Reply:
Not good. Fixed.